src/qt/README.md

```diff
@@ -75,21 +74,23 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for general guidelines. Specifically for
 
 ## Using Qt Creator as IDE
 
-You can use Qt Creator as an IDE. This is especially useful if you want to change
-the UI layout.
-
-Download and install the community edition of [Qt Creator](https://www.qt.io/download/).
-Uncheck everything except Qt Creator during the installation process.
-
-Instructions for macOS:
-
-1. Make sure you installed everything through Homebrew mentioned in the [macOS build instructions](/doc/build-osx.md)
-2. Use `./configure` with the `--enable-debug` flag
-3. In Qt Creator do "New Project" -> Import Project -> Import Existing Project
-4. Enter "bitcoin-qt" as project name, enter src/qt as location
-5. Leave the file selection as it is
-6. Confirm the "summary page"
-7. In the "Projects" tab select "Manage Kits..."
-8. Select the default "Desktop" kit and select "Clang (x86 64bit in /usr/bin)" as compiler
-9. Select LLDB as debugger (you might need to set the path to your installation)
-10. Start debugging with Qt Creator (you might need to the executable to "bitcoin-qt" under "Run", which is where you can also add command line arguments)
+[Qt Creator](https://www.qt.io/product/development-tools) is a powerful tool which packages a UI Designer tool (Qt Designer) and a C++ IDE into one application. This is especially useful if you want to change the UI layout.
+
+#### Download Qt Creator
+On Linux and macOS, Qt Creator can be installed through your package manager. Alternatively, you can download a binary from the [Qt Website](https://www.qt.io/download/).
+During the installation process, uncheck everything except for `Qt Creator`.
+
+#### Setup Qt Creator
+1. Make sure you've installed all dependencies specified in your systems build instructions
+2. Follow the compile instructions for your system, run `./configure` with the `--enable-debug` flag
+3. Start Qt Creator. At the start page, do: `New` -> `Import Project` -> `Import Existing Project`
+5. Enter `"bitcoin-qt"` as the Project Name, enter `"{repo}/src/qt"` as Location (replace `{repo}` with the actual repo path)
+6. Check over the file selection, you may need to select the `"forms"` directory (necessary if you intend to edit *.ui files)
+7. Confirm the `Summary` page
+8. In the `Projects` tab, select `Manage Kits...`
+ - Under `Kits`: select the default "Desktop" kit
+ - Under `Compilers`: select an appropriate compiler. (on macOS select `"Clang (x86 64bit in /usr/bin)"`)
+ - Under `Debuggers`: select `"LLDB"` as debugger (you might need to set the path to your LLDB installation)
```

Comment:
I guess this is Mac specific (clang in /usr/bin/ as well as `LLDB`). I would probably keep this as "Mac" instructions (Linux/Win) people might need to adapt but they are not blinded by not telling them its made for Mac.

Reply:
Done in f6584d9c84f4d840dd5cb49f7dcc9984e9a16304